StevesSixLeggedSprinters/BangazonAPI
# fixed the migrations folder

## Status
Choose:
**READY**

## Migrations
NO

## Description
I am creating an issue ticket for this as well, but the Migrations Folder is now ignored so that you can re-create on pull.   

## Todos


## Deploy Notes
Migrations folder no longer tracked

## Steps to Test or Reproduce
just pull down with the following. 

```sh
git fetch --all
git branch -avv (see all branches)
git checkout kc-testing
git pull origin kc-testing
```

1. 

## Impacted Areas in Application
List general components of the application that this PR will affect:
Your Migration Folder, you will need to re-run the dotnet ef update database 

*
